scripts/west_commands/runners/jlink.py

```diff
@@ -243,6 +247,7 @@ def do_run(self, command, **kwargs):
                        '-speed', self.speed,
                        '-device', self.device,
                        '-silent',
+                       '-endian', self.endian,
```

Comment:
Can we set this automatically based on `CONFIG_BIG_ENDIAN`/`CONFIG_LITTLE_ENDIAN`?

Reply:
Ah yes, that is a good idea. What is the best way of doing that? Using `zcmake.CMakeCache`, or is there another way?

Reply:
You should be able to use `self.build_conf.getboolean('CONFIG_BIG_ENDIAN')` within the runner. Check the other runners for examples on how this can be done.